charts/openstack-cloud-controller-manager/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: Openstack Cloud Controller Manager Helm Chart
 icon: https://object-storage-ca-ymq-1.vexxhost.net/swift/v1/6e4619c416ff4bd19e1c087f27a43eea/www-images-prod/openstack-logo/OpenStack-Logo-Vertical.png
 home: https://github.com/kubernetes/cloud-provider-openstack
 name: openstack-cloud-controller-manager
-version: 2.29.0-alpha.4
+version: 2.29.0-alpha.5
```

Comment:
probably conflicts with https://github.com/kubernetes/cloud-provider-openstack/pull/2446/files#diff-9885a0a93aed735b5f9e5eb3bada39bf4b2554b6e776c215d181d61008790c15R7, happy to update as needed